Source/Core/Common/CMakeLists.txt

```diff
@@ -117,6 +117,10 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
   set(LIBS ${LIBS} dl)
 endif()
 
+if(OPROFILE_FOUND)
```

Comment:
Could you do this in a more-modern-CMake way, like:

```cmake
add_dolphin_library(common "${SRCS}" "${LIBS}")

if(OPROFILE_FOUND)
  target_link_libraries(common PRIVATE ${OPROFILE_LIBRARIES})
endif()
```

? Then the various includes/other libs/whatever don't get unnecessarily propagated to libraries that use Common.